From c1677527af3f7afd48ebbe1b2338ac822f64c524 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sat, 19 Jun 2021 19:27:09 +0200
Subject: [PATCH] fix: fix regex in test_get_python_version for Python 3.10

Fix the regular expression in test_get_python_version to permit
the minor version to contain more than one digit.  This fixes the test
on Python 3.10.
---
 tests/test_cmaker.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_cmaker.py b/tests/test_cmaker.py
index d63059c..52d0028 100644
--- a/tests/test_cmaker.py
+++ b/tests/test_cmaker.py
@@ -21,7 +21,7 @@ from . import _tmpdir, get_cmakecache_variables
 
 
 def test_get_python_version():
-    assert re.match(r'^[23](\.?)[0-9]$', CMaker.get_python_version())
+    assert re.match(r'^[23](\.?)\d+$', CMaker.get_python_version())
 
 
 def test_get_python_include_dir():
-- 
2.32.0

